src/deadline/client/cli/_common.py

```diff
@@ -182,3 +189,42 @@ def _cli_object_repr(obj: Any):
     # strings to end with "\n".
     obj = _fix_multiline_strings(obj)
     return deadline_yaml_dump(obj)
+
+
+class _ProgressBarCallbackManager:
```

Comment:
Re-using from other bundle groups.

Reply:
Its common util, but still internal to this package. Its not public for external users so `_` should stay.